doc/source/whatsnew/v0.17.1.txt

```diff
@@ -25,6 +25,7 @@ Enhancements
   objects for the ``filepath_or_buffer`` argument. (:issue:`11033`)
 - ``DataFrame`` now uses the fields of a ``namedtuple`` as columns, if columns are not supplied (:issue:`11181`)
 - Improve the error message displayed in :func:`pandas.io.gbq.to_gbq` when the DataFrame does not match the schema of the destination table (:issue:`11359`)
+- Prevent adding new attributes to the accessors .str, .dt and .cat (:issue:`10673`)
```

Comment:
I was actually thinking it should go to bugfix: currently you can't get the value back, so the concept is broken and we only make the error more explicit.